IntegrationTests/BTCardClient_IntegrationTests.swift

```diff
@@ -5,8 +5,10 @@ import XCTest
 class BTCardClient_IntegrationTests: XCTestCase {
 
     func testTokenizeCard_whenCardHasValidationDisabledAndCardIsInvalid_tokenizesSuccessfully() {
-        let apiClient = BTAPIClient(authorization: BTIntegrationTestsConstants.sandboxTokenizationKey)!
-        let cardClient = BTCardClient(apiClient: apiClient)
+        var apiClient = BTAPIClient(authorization: BTIntegrationTestsConstants.sandboxTokenizationKey)!
+        var cardClient = BTCardClient(authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn")
```

Comment:
Should these be using the same authorization?

Reply:
ahh yep, ty addressed in deba6e188867bd8c726b3073b4a4a2e0ab602318